docs/source/users/kernel-envs.md

```diff
@@ -75,7 +75,9 @@ There are several supported `KERNEL_` variables that the Enterprise Gateway serv
     cases, Enterprise Gateway will honor that value.  However, when provided,
     it is the user's responsibility that KERNEL_POD_NAME is unique relative to
     any pods in the target namespace.  In addition, the pod must NOT exist -
-    unlike the case if KERNEL_NAMESPACE is provided.
+    unlike the case if KERNEL_NAMESPACE is provided. The KERNEL_POD_NAME can
+    also be provided as a jinja2 template (e.g "{{ kernel_id }}") string which
```

Comment:
I think it would be good to show a practical example rather than just kernel-id. Could you please add a string prefix like "my-kernel" or something?